build-system/tasks/check-links.js

```diff
@@ -12,6 +12,10 @@ const {log, logLocalDev} = require('../common/logging');
 const LARGE_REFACTOR_THRESHOLD = 20;
 const GITHUB_BASE_PATH = 'https://github.com/ampproject/amphtml/blob/main/';
 
+// Certain sites do not like the default user agent string.
+const USER_AGENT =
+  'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36';
```

Comment:
I wonder if there's some npm package that would pin this to Chrome latest, instead of us having to manually update this every X months/years whenever this breaks

Reply:
I think https://www.npmjs.com/package/user-agents can work, but these servers primarily use blocklist for `curl`, `node` or other things applications they do not like, rather than blocking specific browser versions. Fixing on a specific version of UA would unlikely fail in a few years. Therefore I would think the cost of adding a dependency would outweigh the benefit.